ApollosProject/apollos-apps
# fix(community): adds safe area padding to non joined community image

## ✏️ Description
- fixes padding issue for non joined community image


## 🔬 To Test
1. Go to Community tab
2. View community you have not joined yet
3. Top padding on image should be below notch

## 📸 Screenshots
![image](https://user-images.githubusercontent.com/7557597/215583410-43b6de02-c88e-42a7-9d71-5b95cc0fe4e4.png)



## ⚠️ To-do Before Merge



## 📝 Checklist
